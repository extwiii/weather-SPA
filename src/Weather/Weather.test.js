import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Weather from './index';

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() });

it('App component renders correctly', () => {
  const wrapper = shallow(<Weather />);
  expect(wrapper).toMatchSnapshot();
});

it('App component renders correctly with props', () => {
  const wrapper = shallow(<Weather title="Now" icon="myIcon" temperature={32} />);
  expect(wrapper).toMatchSnapshot();
});
