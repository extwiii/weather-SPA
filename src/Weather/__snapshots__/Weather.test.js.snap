// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<div>
  <h3
    style={
      Object {
        "color": "#48F",
      }
    }
  >
    Loading
  </h3>
  <img
    alt=""
    src="https://openweathermap.org/img/w/.png"
  />
  <p>
    0
     °C
  </p>
</div>
`;

exports[`App component renders correctly with props 1`] = `
<div>
  <h3
    style={
      Object {
        "color": "red",
      }
    }
  >
    Now
  </h3>
  <img
    alt="myIcon"
    src="https://openweathermap.org/img/w/myIcon.png"
  />
  <p>
    32
     °C
  </p>
</div>
`;
